res/skins/Tango/buttons/btn_beats_set_bar_beat.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
AFAIK new files need to be added to `mixxx.qrc`.

Reply:
Ni idea of what is mixxx.qrc, seems like a Qt thing. I'm quite illiterate on Qt, but learning a lot here.
I can't see any reference to other skin XML files. So I suppose that is not needed.
If someone can explain me how to fill it I can do it if needed.
Thanks for reviewing my PR :)

Reply:
Not in this case.
We ship the skin folder along with Mixxx so the new file will be already available.
The qrc file is used to add recources directly to the binary mixxx file.

Reply:
Good copy.
No change needed.